queue/queue_test.go

```diff
@@ -0,0 +1,995 @@
+package queue
```

Comment:
I can't actually tell what changed because of the file rename. Maybe let's talk in person for a minute.